internal/osutils/autostart/autostart_linux.go

```diff
@@ -55,21 +55,30 @@ func (a *App) Enable() error {
 	return nil
 }
 
-func (a *App) Disable() error {
+func (a *App) Path() (string, error) {
 	enabled, err := a.IsEnabled()
 	if err != nil {
-		return errs.Wrap(err, "Could not check if app autostart is enabled")
+		return "", errs.Wrap(err, "Could not check if app autostart is enabled")
 	}
 	if !enabled {
-		return nil
+		return "", nil
```

Comment:
This feels inappropriate for `.Path()`, why should it care whether it is enabled? You could want to use this to set up the path.